src/lua/BabblerBomb/EnumUtils.lua

```diff
@@ -0,0 +1,79 @@
+EnumUtils = {}
```

Comment:
This will overwrite any other version of the EnumUtils, might want to scope it properly ;)